app/models/elastic_custom_client.rb

```diff
@@ -0,0 +1,19 @@
+require 'multi_json'
+require 'faraday'
+require 'elasticsearch/api'
+
+class ElasticCustomClient
+  include Elasticsearch::API
+
+  CONNECTION = ::Faraday::Connection.new url: ENV["ELASTICSEARCH_URL"]
+
+  def perform_request(method, path, params, body)
+    puts "--> #{method.upcase} #{path} #{params} #{body}"
+
+    CONNECTION.run_request \
+      method.downcase.to_sym,
+      path,
+      ( body ? MultiJson.dump(body): nil ),
```

Comment:
Surrounding space missing for operator `:`.<br>Space inside parentheses detected.